### PR TITLE
feat(generator): schematic to generate an operator

### PR DESCRIPTION
**Pull request by matthieu-crouzet**
## Related issues

- :bug: Fixes #(issue)
- :rocket: Feature #(issue)

### CODE REVIEW

**Review by fpaul-1A — COMMENTED** [review]
missing update on the `readme.md`

File: packages/@o3r/rules-engine/schematics/operator/schema.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import type { SchematicOptionObject } from '@o3r/schematics';
+
+type SupportedSimpleTypes =
```
Comment (cpaulve-1A) [inline packages/@o3r/rules-engine/schematics/operator/schema.ts:3]: do you really need to declare it once again?

**Review by cpaulve-1A — APPROVED** [review]

**Review by fpaul-1A — APPROVED** [review]

File: packages/@o3r/rules-engine/schematics/operator/index.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { apply, chain, MergeStrategy, mergeWith, move, noop, renameTemplateFiles, Rule, template, url } from '@angular-devkit/schematics';
+import { applyEsLintFix } from '@o3r/schematics';
+import { NgGenerateOperatorSchematicsSchema } from './schema';
```
Comment (kpanot) [inline packages/@o3r/rules-engine/schematics/operator/index.ts:3]:
```suggestion
import type { NgGenerateOperatorSchematicsSchema } from './schema';
```

**Comment by kpanot** [comment]
Should also include the VSCode extensions to use the schematics

**Comment by matthieu-crouzet** [comment]
> Should also include the VSCode extensions to use the schematics (or a link to the task to create it)

https://github.com/AmadeusITGroup/otter/issues/704

**Review by kpanot — APPROVED** [review]

**Review by fpaul-1A — APPROVED** [review]

**Pull request merged**